projects/Amlogic/filesystem/usr/share/bootloader/update.sh

```diff
@@ -98,6 +98,14 @@ if [ -f $SYSTEM_ROOT/usr/share/bootloader/boot.ini ]; then
   cp -p $SYSTEM_ROOT/usr/share/bootloader/boot.ini $BOOT_ROOT/boot.ini
 fi
 
+if [ -f $SYSTEM_ROOT/usr/share/bootloader/user.ini ]; then
```

Comment:
Please merge this with the case above it.

Reply:
Can do.